CHANGES

```diff
@@ -18,6 +18,7 @@ Bugs fixed
 
 * #6286: C++, allow 8 and 9 in hexadecimal integer literals.
 * #6305: Fix the string in quickstart for 'path' argument of parser
+* LaTeX: Figures in adminitions produced errors (float in float)
```

Comment:
typo: s/adminitions/admonitions

Also "float in float" is not technically the exact LaTeX problem so it should be suppressed.